Docker/Production/entando-full-stack/appbuilder/README.md

```diff
@@ -22,4 +22,11 @@ All the following would be valid values:
 
 ## Exposed ports
 
-`3000`
\ No newline at end of file
+`3000`
```

Comment:
Should this have 5000 in it as well based on the dockerfile?

Reply:
Only 5000. Recent change to NodeJS 'serve'. Fixed. Will remove 3000 from the Dockerfile. Its been on 5000 for a month so it seems they made up their mind.